src/sc/embedder/content/embedder.py

```diff
@@ -438,3 +438,32 @@ class EmbedderVideoJS(BrowserView):
 
     def __call__(self):
         return self.index()
+
+
+@indexer(IEmbedder)
+def searchable_text_indexer(obj):
+    """SearchableText should contain id, title, description, body text,
+    alternate text and keywords.
+    """
+    transformer = ITransformer(obj)
+
+    try:
+        text = transformer(obj.text, 'text/plain')
```

Comment:
Just a minor suggestion, feel free to discard, but this may be just a bit faster and possibly more readable/succinct:

`text = transformer(obj.text, 'text/plain') if getattr(obj, 'text') else ''`

same for `alternate_text`

Reply:
actually, quoting S.O. `decision depends on the probability of exceptional cases.` https://stackoverflow.com/questions/1835756/using-try-vs-if-in-python

Reply:
that's why I prefer to use the try/except approach in this case.